cmd/broker/filter/main.go

```diff
@@ -80,29 +70,44 @@ func main() {
 		log.Fatal("Error building kubeconfig", err)
 	}
 
-	kubeClient := kubernetes.NewForConfigOrDie(cfg)
+	ctx, _ = injection.Default.SetupInformers(ctx, cfg)
+	kubeClient := kubeclient.Get(ctx)
+
+	loggingConfig, err := sharedmain.GetLoggingConfig(ctx)
+	if err != nil {
+		log.Fatal("Error loading/parsing logging configuration:", err)
+	}
+	logger, _ := logging.NewLoggerFromConfig(loggingConfig, "broker_filter")
```

Comment:
The rest of the file expected the desugared logger, do something like:

```suggestion
	sl, _ := logging.NewLoggerFromConfig(loggingConfig, "broker_filter")
	logger := sl.Desugar()
```